index.html

```diff
@@ -43,31 +41,29 @@ <h2 id="quote" class="quote"></h2>
             <div class="game_controls">
                 <div class="input_wrapper">
                     <label for="guess"></label>
-                    <input name="guess" id="guess" type="text">
-                    <button id="submit_button">Submit</button>
-                    <button id="reveal_button">Reveal Answer</button>
-                    <button id="hint_button">Get Hint</button>
+                    <input name="guess" id="guess" class="guess" type="text">
```

Comment:
Hate to be that guy but this self-closing tag needs a / at the end!!!!

Reply:
done!